documentation/views/portfolio.rst

```diff
@@ -5,20 +5,27 @@ Portfolio overview
 ******************
 
 The portfolio overview shows results and opportunities regarding the user's asset portfolio.
-The view serves to quickly identify upcoming opportunities to valorise on flexibility actions.
+The view serves to get an overview over the portfolio's energy status and also to estimate
+the upcoming opportunities of flexible schedules.
```

Comment:
By far most of the page is about reporting on the past, regarding energy prosumption and previous activation of flexibility. I'd stick with describing only those here.

It's just the red highlight that serves as a call to action on upcoming opportunities, which is more of a friendly nudge to remind a user to head to the control page (it is not a quantitative estimate).